nix/sources.json

```diff
@@ -29,10 +29,10 @@
         "homepage": null,
         "owner": "input-output-hk",
         "repo": "cardano-wallet",
-        "rev": "a37c9856b4d96286e3f01a95026c63b986ebbba0",
-        "sha256": "1mg8n58j2mjqhhzjb4p5yp8z06b9arh40pagi9rddil2f3vxzihm",
+        "rev": "b6810e8401b04b1d8a1a142f0b06b47cd6b51186",
```

Comment:
@tomislavhoracek please see if this version is enough: `b64b63fd56dddbe3ff070dea78977f0f15a487c2 `

That's the one we used here: https://github.com/input-output-hk/daedalus/pull/2292/files#diff-b9a0014cf6c4615d1c2e8d921dc3db59b4a68b462788d667ed622331660d17cd

Reply:
@nikolaglumac Withdrawals are not applied to coins selection from that revision.
Your CWB rev - `Feb 15, 2021`
Withdrawals applied in rev- `Feb 18, 2021`

Reply:
Then we can not merge this today.